homeworks/andrey.makarenko_Baksckua/5-arrays/arrays-1.js

```diff
@@ -0,0 +1,15 @@
+const array = [
+    [1, 4, 2, 4, 4, 2, 4, 3, 2, 6, 2, 4, 2, 4, 3, 1, 2, 4, 3],
+    [1, 1, 3, 1, 1, 3, 3, 1, 6, 6, 8, 3, 3, 3, 3, 1, 4, 1, 2],
+    [1, 2, 1, 4, 1, 1, 4, 7, 8, 8, 7, 7, 1, 1, 4, 3, 1, 1, 2],
+    [3, 2, 3, 3, 4, 2, 7, 8, 6, 8, 6, 7, 8, 1, 2, 3, 3, 3, 4],
+    [4, 1, 3, 3, 2, 8, 7, 8, 6, 7, 6, 7, 6, 7, 2, 4, 3, 2, 2],
+    [3, 1, 2, 1, 7, 7, 8, 8, 8, 7, 6, 8, 8, 6, 8, 4, 1, 3, 2],
+    [2, 1, 1, 7, 6, 7, 7, 8, 7, 6, 8, 6, 6, 6, 6, 8, 4, 4, 3],
+    [4, 1, 6, 7, 8, 7, 8, 8, 6, 6, 7, 7, 6, 6, 6, 7, 6, 1, 2],
+    [1, 6, 6, 6, 6, 8, 8, 7, 6, 6, 7, 8, 7, 8, 8, 7, 6, 7, 1],
+    [8, 6, 8, 6, 6, 7, 7, 6, 6, 7, 8, 7, 6, 6, 6, 6, 7, 8, 7],
+];
+
+const arr = array.map((ofArray) => ofArray.map((of) => (of > 4 ? '$' : '*')).join(''));
```

Comment:
1) `arr` is a bad name for the changed array. Could you let me know what you did here? You have remapped the array to have a new array. So let's rename it to a `resultArray`;

2) `ofArray` is a bad name for an array row. By the way, `row` will sound better ;)

3) `of`? What does it mean? If we image this array as a table, we have `rows`, `columns`, and `cells`. So the best name for it would be `cell`. Please, rename it